proposals/NNNN-GetSystemTime.md

```diff
@@ -0,0 +1,64 @@
+# GetSystemTime RPC
+
+* Proposal: [SDL-NNNN](NNNN-GetSystemTime.md)
+* Author: [Aleksandr Stasiuk](https://github.com/AStasiuk)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC GetSystemTime inorder for SDL Core to get accurate system time information which is utilized by the TLS handshake mechanism.
+
+
+## Motivation
+
+Head units can make use of TLS handshake mechanism for security reasons to ensure the connected applications are authentic. This becomes necessary for projection experiences (such as Mobile Navigation) where the app has complete control over the content it displays. To facilitate such security mechanisms SDL Core will need to be provided with accurate time information. Today, SDL Core can get the time information by querying the platform OS which may or may not be accurate.
+This proposal provides a solution to this problem by having SDL Core request the system for time information. The system would then be responsible for providing accurate time related information.
+
+
+## Proposed solution
+
+
+The solution proposes adding a new RPC between SDL and HMI called `GetSystemTime`. This RPC allows SDL Core to query the HMI for the accurate system time.
+
+SDL Core must send the  `GetSystemTime` RPC to HMI for obtaining the system time during Security TLS Handshake.

```

Comment:
SDL Core must send the  `GetSystemTime` RPC to the HMI for obtaining the system time during the Security TLS Handshake.

Reply:
@mrapitis, Fixed in https://github.com/smartdevicelink/sdl_evolution/pull/324/commits/bbe0f4151e53cc0c528ca66fba27f1371e87423f